src/applications/simple-forms/mock-simple-forms-patterns/arrayBuilder/helpers.js

```diff
@@ -51,3 +51,54 @@ export function onNavForwardKeepUrlParams({ goNextPath, urlParams }) {
 export function onNavBackKeepUrlParams({ goPreviousPath, urlParams }) {
   goPreviousPath(urlParams);
 }
+
+/**
+ * Gets the query params from the URL and parses them as an object
+ *
+ * @param {string} [queryString] optional e.g. `?add=true`
+ * @returns {object} e.g. `{ add: 'true' }`
+ */
+export function parseQueryParams(queryString = window.location?.search) {
+  return queryString?.split('?')?.reduce((acc, query) => {
```

Comment:
For the future refactor on this, it might make more sense to remove `?`s (instead of splitting on them) and split on `&`s instead.